install_config/configuring_openstack.adoc

```diff
@@ -40,7 +39,55 @@ Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
 [[openstack-configuring-masters]]
-== Configuring Masters
+== Configuring OpenStack for {product-title} Masters
+
+You can set OpenStack configuration on your {product-title} masters two ways:
+
+- xref:openstack-configuring-masters-ansible[using Ansible and the advanced installation tool]
+- xref:openstack-configuring-masters-manually[manually by modifying the *_master-config.yaml_* file]
+
+[[openstack-configuring-masters-ansible]]
+=== Configuring AWS for {product-title} with Ansible
```

Comment:
This probably needs a note that explains this sets up and configures both masters and nodes.